libs/metagen/src/lib.rs

```diff
@@ -0,0 +1,148 @@
+// Copyright Metatype OÜ, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+#[allow(unused)]
+mod interlude {
```

Comment:
:) where does this name come from?

Reply:
It's a play on the word `prelude` but in this case, only for crate private i.e. "**inter**nal" consumption. The actual English word means "intermission" which is not exactly related I suppose.